lib/plugins/features/movies/controller.js

```diff
@@ -11,10 +11,19 @@ exports.create = (payload) => {
   });
 };
 
-exports.get = (query) => {
-
-  return Movie.where('release_year', '<', query.release_year).fetchAll()
-  .then((movies) => {
-    return movies.toJSON();
-  });
+exports.list = (query) => {
+  return Movie
```

Comment:
Super nit: while it might functionally work the same way, we prefer to work with instance methods on the model instead of class methods. This means using `new Movie()` instead of just `Movie`.